### ISSUE TITLE
Support freedesktop/portal background API

### ISSUE THREAD

**Issue by sonnyp** [dino/dino#1476]
# Design

Dino should autostart and run in the background by default so users don't miss messages / have to think of opening the app.

# Implementation

https://flatpak.github.io/xdg-desktop-portal/#gdbus-org.freedesktop.portal.Background

Gtk doesn't have an API for this (yet?) but libportal can be used

* https://libportal.org/method.Portal.request_background.html
* https://libportal.org/method.Portal.set_background_status.html

# Related

* https://github.com/dino/dino/issues/98
* https://github.com/dino/dino/issues/833
* https://github.com/dino/dino/issues/658
* https://github.com/dino/dino/issues/299

**Comment by chpio** [comment]
Are using a desktop, that supports autostarting apps? Gnome for example has its tweak tool, that can be used to start any app you want on login.

**Comment by craftyguy** [comment]
@chpio I don't think this is quite the same thing, and Dino can be run on non-desktop systems (like a real phone running Linux!) Having the GUI opened just to receive notifications isn't great because it can be accidentally closed by the user and it constantly takes up space somewhere in the UI.

**Comment by rmader** [comment]
Made a PoC for it: https://github.com/dino/dino/pull/1826
